### ISSUE TITLE
Save actions are saved in a single line at the moment.

### ISSUE THREAD

**Issue by simonharrer** [JabRef/jabref#1419]
JabRef 3.4dev--snapshot--2016-05-19--master--ed638e2
windows 10 10.0 amd64 
Java 1.8.0_92

Save actions are saved in a single line at the moment. 

```diff
@Comment{jabref-meta: saveActions:enabled;
-pages[normalize_page_numbers]
-month[normalize_month]
-author[normalize_names]
-booktitle[ordinals_to_superscript]
-;}
+pages[normalize_page_numbers]month[normalize_month]author[normalize_names]booktitle[ordinals_to_superscript];}
```

Expected:

```diff
@Comment{jabref-meta: saveActions:enabled;
pages[normalize_page_numbers]
month[normalize_month]
author[normalize_names]
booktitle[ordinals_to_superscript]
;}
```

**Comment by lenhard** [comment]
JabRef 3.4dev--snapshot--2016-05-23--master-- 4aa51a3
windows 7 6.1 amd64 
Java 1.8.0_92

I just tried this in current master and everything works as expected. The save actions are written to file as you outline under "Expected". Can someone else reproduce this issue? Might there be something strange going on with your newline configuration?

**Comment by simonharrer** [comment]
I do even get strange results when disabling the save actions (from enabled to disabled):

```diff
-@Comment{jabref-meta: saveActions:enabled;
-pages[normalize_page_numbers]
+@Comment{jabref-meta: saveActions:disabled;pages[normalize_page_numbers]
 month[normalize_month]
 author[normalize_names]
 booktitle[ordinals_to_superscript]
 title[protect_terms]
 ;}
```

**Comment by lenhard** [comment]
At least I can confirm the missing line break when disabling the save actions!

**Comment by lenhard** [comment]
The last point should be fixed with a84502158b8d6be2feb9c2b8b783d41f78c7d8d6 Please re-check on current master.

I still cannot reproduce the other problems, though.

**Comment by simonharrer** [comment]
maybe it was an old version, I cannot reproduce it anymore right now. Hence, I close it.

**Comment by simonharrer** [comment]
Hm, I have this issue again.

**Comment by lenhard** [comment]
Ok, I can reproduce that!

**Comment by lenhard** [comment]
@simonharrer: So far the save actions only went through the formatter when the respective panel was being opened. If not, the newlines would be eliminated by the parsing of the meta data. I fixed that in  2a979fd on master: The formatter string is now properly loaded and set in the `MetaData`. 

Please verify that it works for you.

**Comment by simonharrer** [comment]
LGTM
